src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2173,6 +2191,22 @@
   <data name="Format_BadDateTime" xml:space="preserve">
     <value>String '{0}' was not recognized as a valid DateTime.</value>
   </data>
+  <data name="Format_BadDateOnly" xml:space="preserve">
+    <value>String '{0}' was not recognized as a valid DateOnly.</value>
+    <comment>{Locked="DateOnly"}</comment>
+  </data>
+  <data name="Format_BadTimeOnly" xml:space="preserve">
+    <value>String '{0}' was not recognized as a valid TimeOnly.</value>
+    <comment>{Locked="TimeOnly"}</comment>
+  </data>
+  <data name="Format_DateOnlyContainsNoneDateParts" xml:space="preserve">
+    <value>String '{0}' contains parts which are not specific to the DateOnly.</value>
+    <comment>{Locked="DateOnly"}</comment>
+  </data>
+  <data name="Format_TimeOnlyContainsNoneTimeParts" xml:space="preserve">
+    <value>String '{0}' contains parts which are not specific to the TimeOnly.</value>
```

Comment:
Standing on my soapbox for a moment, to my eyes this would read a lot better if it were "which are not specific to the TimeOfDay".

Runs and ducks for cover...

Reply:
ok, I'll use `which are not specific to the time of the day`. I want avoid teh confusion if we use `TimeOfDay`.

Reply:
You don't need to change anything.  Sorry, I was just poking at the type name again; I realize we're not going to change it at this point.

Reply:
oh, got it. thanks for clarifying :-) I'll not change it then.